tests/helper/helper_cluster.go

```diff
@@ -13,36 +13,36 @@ import (
 	"github.com/onsi/gomega/gexec"
 )
 
-type OcRunner struct {
-	// path to oc binary
+type CliRunner struct {
```

Comment:
Why renaming this to generic `CliRunner`? 
There are functions in this that are specific just for `oc`.

Reply:
We have some test where we have to switch runner according to the cluster type i.e Kubernetes or openshift. In that case handling the kubectl and oc runner will become tough. So, thinking from that prospective and reusability of code I was thinking to generalise it for kubectl and oc. 

Yup, like you mentioned function like SwitchProject... won't work for sure, however we can handle those through if...else statement or may be through switch statment depending upon which client is being used. May be in another issue i can analysze those incompatibility and fix those in a separate pr.

Reply:
I would suggest we don’t rename to CliRunner

Reply:
ok, let me revisit the change again, also let me find a better way to implement it. Till then i am holding the pr

Reply:
> ok, let me revisit the change again, also let me find a better way to implement it. Till then i am holding the pr

+1

Reply:
> I would suggest we don’t rename to CliRunner

@girishramnani @kadel I have done some modification with common interface for all the cli. Can you please look into this once.